src/components/Calculator.css

```diff
@@ -0,0 +1,41 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin-top: 10%;
+  margin-left: 20%;
+}
+
+button {
+  appearance: none;
+  border: none;
+  outline: none;
+  color: #000;
+  font-size: 20px;
+  padding: 16px;
+  cursor: pointer;
+  transition: 0.4s;
+  background-color: #e0e0e0;
+}
+
+.orange {
+  background-color: #f5913e;
+}
+
+.display {
+  background-color: #858694;
+  width: 65.4%;
+  color: #fff;
+  margin-left: 12px;
+}
+
+span {
+  padding-left: 100%;
+  font-size: 20px;
+}
+
+button:hover {
+  opacity: 0.9;
+}
```

Comment:
- [ ] [REQUIRED] In the image below, I've highlighted the `red lines` between you and the incorrect differences in the `Correct` design. Please space your buttons apart like the `Correct` design! 

![image](https://user-images.githubusercontent.com/63915024/190503088-182e2757-5769-4d88-b585-70ae9229c848.png)

Reply:
Thank you for your review.
But I didn't clearly understand what you want me to change can you explain more thanks.